src/index.js

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
 import { render } from 'react-dom';
+import _ from 'lodash';
+import PhotoGallery from 'photo-gallery';
+
+let photos = _.map (_.range (20), function (num) {
+  return {
+    path: 'http://loremflickr.com/640/480?random=' + num
```

Comment:
That would be a slick way of doing it. I was just using the example code that was provided.